README.md

```diff
@@ -57,8 +72,10 @@ This function returns `true` when the call is successful. Otherwise it returns
 `nil` and a string describing the error.
 
 
+[Back to TOC](#table-of-contents)
```

Comment:
A very minor comment: there's two line jumps before this link. Really not a blocker.